spyder/plugins/completion/kite/providers/document.py

```diff
@@ -116,10 +116,14 @@ def request_hover(self, params):
     @handles(LSPRequestTypes.DOCUMENT_HOVER)
     def process_hover(self, response):
         # logger.debug(response)
-        report = response['report']
-        text = report['description_text']
-        if len(text) == 0:
+        if response:
```

Comment:
Perhaps we should change this to

    if response is not None:

to be more explicit.